### ISSUE TITLE
Make sentence method

### ISSUE THREAD

**Issue by accraze** [accraze/python-ia-markov#18]
Right now we don't have a great API for retrieving a sentence.... we have to call a method our object's `model` which isn't great.

We should add a `make_sentence` method to `MarkovModel` so we can do something like the following:
```
from ia_markov import MarkovModel

m = MarkovModel()
m.train_model('FuturistManifesto')
m.model.make_sentence()  # old way
m.make_sentence()  # new way
```

**Comment by Abelarm** [comment]
i'd like to work on this.

**Comment by accraze** [comment]
Closed by #22
